sdk/src/builder.rs

```diff
@@ -80,6 +79,8 @@ pub struct ManifestDefinition {
     #[serde(default = "default_instance_id")]
     pub instance_id: String,
 
+    /// An optional ResourceRef to an image.  Must be available when the manifest is signed.
```

Comment:
This needs to say something about being a thumbnail image that represents the asset.

Reply:
Addressed.